build_scripts/webpack-rules.js

```diff
@@ -23,7 +23,11 @@ const path = require('path')
 module.exports = [
   {
     test: /\.(js|jsx)$/,
-    exclude: /(node_modules)|(cypher-codemirror)|(test_utils)|(dist)/,
+    include: [
```

Comment:
I'm a bit worried that this might break something but from my tests it all seems fine.
We should verify each step in the build pipeline after this is merged.